src/main/java/org/opensearch/ad/ExecuteADResultResponseRecorder.java

```diff
@@ -50,11 +49,12 @@
 import org.opensearch.timeseries.common.exception.TimeSeriesException;
 import org.opensearch.timeseries.model.FeatureData;
 import org.opensearch.timeseries.model.IntervalTimeConfiguration;
+import org.opensearch.timeseries.util.DiscoveryNodeFilterer;
 
 public class ExecuteADResultResponseRecorder {
     private static final Logger log = LogManager.getLogger(ExecuteADResultResponseRecorder.class);
 
-    private AnomalyDetectionIndices anomalyDetectionIndices;
+    private ADIndexManagement anomalyDetectionIndices;
```

Comment:
rename?

Reply:
rename anomalyDetectionIndices , to be clear.

Reply:
I will have a different one for forecast. Their functionalities are different.